db/migrate/20221102032844_add_users_and_posts_ref_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsersAndPostsRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
* As mentioned above 🔼 Kindly ensure your foreign keys are the same as in the ERD diagram currently yours says `users_id` instead of `author_id`